### PR TITLE
Implemented state iterations in the eventbrite-API.py

### PR DESCRIPTION
**Pull request by matthew-r-sutton**
1) Implemented API request iterations by state.
2) Altered params so that the date range is 30 days from the current date.

### CODE REVIEW

File: scripts/eventbrite-API.py [hunk 1]
```diff
@@ -17,82 +17,121 @@
...
-# There's going to be overlap between states with that approach,
-# but we can eliminate duplicates in the DB afterwards.
-####
-# us_states = ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE",
-#              "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS",
-#              "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS",
-#              "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY",
-#              "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC",
-#              "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY"]
+us_states = [
```
Comment (pilimayora) [inline scripts/eventbrite-API.py]: [SC]
1. Call the variable something more representative (`us_states_and_view_ports`)
2. Separate this data into separate file for neatness.

**Review by pilimayora — APPROVED** [review]
Looks good!
Let me know if you have time to do the pagination bit too. Otherwise I'll take over, no problem at all.

File: scripts/eventbrite-API.py (hunk 1 above)
Comment (matthew-r-sutton) [reply scripts/eventbrite-API.py]: Added SC's

**Comment by matthew-r-sutton** [comment]
Newest commit iterates through states, dates, and pages.

Still trying to figure out how to speed the script up, as it takes about 1.25-1.45 seconds to process each event--meaning 5,000 events would take around 1.75 hours. Will discuss more tomorrow at our meeting.

**Comment by matthew-r-sutton** [comment]
So I omitted date iterations. Doing so (for WI) proved to be about 40% quicker--hopefully the same is true for larger states!

**Comment by pilimayora** [comment]
Nice work! 
Since we are not iterating through days (good work checking speed), we could simplify the date_range bit part of the script, but it's definitely not necessarily and I wouldn't block this PR for that. 
Merging!

**Pull request merged**